### PR TITLE
Fix the get-started tutorial when a server is not available - VPN-3552

### PR DESCRIPTION
**Pull request by bakulf**
This is a fix in case:
- 1 city from the first country is unavailable
- all the cities from the first country are unavailable

It's not a fix in case, the first and the second countries do not have cities available. If we want to support this scenario, we need to implement the scrolling. And this becomes out-of-scope for 2.13/2.14.

### CODE REVIEW

File: addons/tutorial_01_get_started/manifest.json [hunk 1]
```diff
@@ -20,8 +20,8 @@
           "op": "vpn_off"
         },{
           "op": "vpn_location_set",
-          "exitCountryCode": "at",
-          "exitCity": "Vienna",
```
Comment (MattLichtenstein) [inline addons/tutorial_01_get_started/manifest.json:24]: Why was this changed?

Comment (bakulf) [reply addons/tutorial_01_get_started/manifest.json:24]: because Austria is the second country in the list (at least in most of the languages). And if Australia is not usable, we switch to the second one, which must be:
- not opened
- not already selected

Comment (MattLichtenstein) [reply addons/tutorial_01_get_started/manifest.json:24]: would it be feasible to add a check to the `hasAvailableCities` property to make sure it has at least one unselected city, and then use that to query for the first country with `hasAvailableCities=true`?

Comment (bakulf) [reply addons/tutorial_01_get_started/manifest.json:24]: Yes, but we do not support scrolling in tutorial yet...

**Review by MattLichtenstein — APPROVED** [review]
